src/graph/executor/query/UnwindExecutor.cpp

```diff
@@ -26,7 +26,7 @@ folly::Future<Status> UnwindExecutor::execute() {
     std::vector<Value> vals = extractList(list);
     for (auto &v : vals) {
       Row row;
-      if (!emptyInput) {
+      if (!unwind->fromPipe() && !emptyInput) {
```

Comment:
How to support multiple variable?
This may break the variable passing ability of ngql's pipe.

Reply:
Unwind does not support multiple variables  in parser